doc/api/path.markdown

```diff
@@ -94,9 +103,13 @@ Windows examples:
 
     path.isAbsolute('//server')  // true
     path.isAbsolute('C:/foo/..') // true
-    path.isAbsolute('bar\\baz')   // false
+    path.isAbsolute('bar\\baz')  // false
     path.isAbsolute('.')         // false
 
+*Note:* If the path string passed as parameter is a zero-length string,
+        unlike other path module functions, zero-length string will be used
+        as-is and `false` will be returned.
```

Comment:
IIRC the behaviour varies on [win32](https://github.com/nodejs/io.js/blob/master/lib/path.js#L212-L215) and [posix](https://github.com/nodejs/io.js/blob/master/lib/path.js#L470-L472).